src/category_theory/limits/shapes/terminal.lean

```diff
@@ -127,25 +127,77 @@ variable (C)
 A category has a terminal object if it has a limit over the empty diagram.
 Use `has_terminal_of_unique` to construct instances.
 -/
-abbreviation has_terminal := has_limits_of_shape (discrete pempty : Type v) C
+abbreviation has_terminal := has_limits_of_shape (discrete.{v₁} pempty) C
 /--
 A category has an initial object if it has a colimit over the empty diagram.
 Use `has_initial_of_unique` to construct instances.
 -/
-abbreviation has_initial := has_colimits_of_shape (discrete pempty : Type v) C
+abbreviation has_initial := has_colimits_of_shape (discrete.{v₁} pempty) C
+
+section univ
+
+variables (X : C) {F₁ : discrete.{w} pempty ⥤ C} {F₂ : discrete.{w'} pempty ⥤ C}
+
+/-- Being terminal is independent of the empty diagram, its universe, and the cone over it,
+    as long as the cone points are isomorphic. -/
+def is_limit_empty_cones {c₁ : cone F₁} (hl : is_limit c₁)
+  (c₂ : cone F₂) (hi : c₁.X ≅ c₂.X) : is_limit c₂ :=
+{ lift := λ c, hl.lift ⟨c.X, by tidy⟩ ≫ hi.hom,
+  fac' := λ _ j, j.elim,
+  uniq' := λ c f _, by { erw ← hl.uniq ⟨c.X, by tidy⟩ (f ≫ hi.inv) (λ j, j.elim), simp } }
+
+/-- Independence of cone as an equivalence. -/
+def is_limit_empty_cones_equiv (c₁ : cone F₁) (c₂ : cone F₂) (h : c₁.X ≅ c₂.X) :
+  is_limit c₁ ≃ is_limit c₂ :=
```

Comment:
This documentation comment does not make sense outside of this file
(for example in the hover-over tool tip).

The same for the dual version.